dynamic-layers/multimedia-layer/recipes-multimedia/rpidistro-vlc/rpidistro-vlc_3.0.12.bb

```diff
@@ -51,8 +51,10 @@ EXTRA_OECONF = "\
 
 PACKAGECONFIG ?= "\
     ${@bb.utils.contains('DISTRO_FEATURES', 'x11', 'x11', '', d)} \
+    ${@bb.utils.contains('DISTRO_FEATURES', 'wayland', 'wayland', '', d)} \
```

Comment:
This looks like another commit to me. Am I missing something?

Reply:
Closing this as well opening another that updates the version of VLC used. Hopefully contain better explanation for changes made.

Reply:
Much better. Cheers!